### PR TITLE
Bottom action bar

### PR DESCRIPTION
**Pull request by ashilkn**
Note : There are some commented out lines of code, will clear them before merging to master. Need them for future context.

https://user-images.githubusercontent.com/77285023/204990851-7f17ebe8-1352-4f8f-b045-95a2c5177b7b.mov

BlurMenuItem component : In Figma the BlurMenuItem includes the 'Cancel' button. But in code, the 'Cancel' button doesn't use the component. Kept it like that to keep it simple. The BlurMenuItem component will be used for actions for the file like move, add, remove etc. Haven't used this component in code yet.

blurMenuItemLight

blurMenuItemDark

### CODE REVIEW

File: lib/ui/viewer/gallery/collection_page.dart [hunk 1]
```diff
@@ -57,38 +78,76 @@ class CollectionPage extends StatelessWidget {
...
       ),
       body: Stack(
         alignment: Alignment.bottomCenter,
         children: [
           gallery,
-          GalleryOverlayWidget(
-            appBarType,
-            _selectedFiles,
-            collection: c.collection,
+          // GalleryOverlayWidget(
```
Comment (vishnukvmd) [inline lib/ui/viewer/gallery/collection_page.dart:106]: We can remove this?

File: lib/ui/components/bottom_action_bar/bottom_action_bar_widget.dart [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+    final textTheme = getEnteTextTheme(context);
+    //todo : restrict width of column
+    return ClipRRect(
+      child: BackdropFilter(
+        filter: ImageFilter.blur(sigmaX: blurBase, sigmaY: blurBase),
+        child: Container(
+          color: colorScheme.backdropBase,
+          padding: const EdgeInsets.only(
+            top: 4,
+            // bottom: !_expandableController.expanded &&
```
Comment (vishnukvmd) [inline lib/ui/components/bottom_action_bar/bottom_action_bar_widget.dart:42]: Can remove this?

**Review by vishnukvmd — APPROVED** [review]

File: lib/ui/components/bottom_action_bar/bottom_action_bar_widget.dart (hunk 2 above)
Comment (ashilkn) [reply lib/ui/components/bottom_action_bar/bottom_action_bar_widget.dart:42]: This too.

File: lib/ui/viewer/gallery/collection_page.dart (hunk 1 above)
Comment (ashilkn) [reply lib/ui/viewer/gallery/collection_page.dart:106]: I had left these intentionally, had mentioned in the description. 
"Note : There are some commented out lines of code, will clear them before merging to master. Need them for future context."

**Pull request merged**